### ISSUE TITLE
Training Issue

### ISSUE THREAD

**Issue by Doruk-Dilmen** [MaximIntegratedAI/ai8x-training#141]
When I use launch.json, such a like that;
![first](https://user-images.githubusercontent.com/91119666/164388009-b15791c8-7096-4f6d-a6df-b50a767f0db7.png)

Early epochs accuracy are high but after that accuracy decreases. For instance 199 epoch:
![second](https://user-images.githubusercontent.com/91119666/164388515-df7a533a-9bc0-4e81-b1c9-5666efe5251c.png)

200 epoch:
![third](https://user-images.githubusercontent.com/91119666/164388566-454dd53e-b69e-4341-9ebb-f5e76c330b82.png)

Why does the confusion matrix come out like this? Could you give an idea?

**Comment by rotx-maxim** [comment]
Could you provide a few more details please?
- Does it work without launch.json?
- If not, are you using the existing KWS20, or your own dataset?
- If yes, is the virtual environment activated, and does the program find the correct schedule files?
- Have you tried launching a shell script (such as the ones provided in the "scripts" folder instead of running train.py) directly?

**Comment by Doruk-Dilmen** [comment]
- Yes, It works without launch.json. It can work with scripts.
- Yes, I am using KWS20 dataset
- Yes, I followed the installation steps on your web site => https://github.com/MaximIntegratedAI/ai8x-training#installation
- I think this problem is hapenning because of learning rate. When I set lr = 0.001, there no such a problem. But, this is the first time I see such a problem like that.

**Comment by Jake-Carter** [comment]
Hi @Doruk-Dilmen,

I agree, I think this may have to do more with the learning rate than the VS Code launch profile.  A learning rate of 0.01 is too high for this model.  Training always diverged if I used --lr 0.01 in my own testing.

[Here ](https://thelaziestprogrammer.com/sharrington/math-of-machine-learning/gradient-descent-learning-rate-too-high)is a decent article visualizing the subject.

The recommended learning rate for this model is 0.001, and our [`train_kws20.sh`](https://github.com/MaximIntegratedAI/ai8x-training/blob/develop/scripts/train_kws20.sh) script uses that learning rate.

I will go ahead and close this ticket with the recommendation to use --lr 0.001
